Gopkg.toml

```diff
@@ -4,24 +4,58 @@
 
 [[constraint]]
   name = "github.com/docker/cnab-to-oci"
-  version = "v0.2.0-beta1"
+  source = "github.com/eunomie/cnab-to-oci"
+  branch = "push-images"
+  #version = "v0.2.0-beta1"
 
 [[constraint]]
   name = "github.com/in-toto/in-toto-golang"
    source = "github.com/radu-matei/in-toto-golang"
    branch = "inspections-dir"
 
-[[override]]
-  name = "github.com/containerd/containerd"
-  revision = "7c1e88399ec0b0b077121d9d5ad97e647b11c870"
+[[constraint]]
+  name = "github.com/docker/cli"
+  revision = "f95ca8e1ba6c22c9abcdbf65e8dcc39c53958bba"
 
 [[override]]
-  name = "github.com/containerd/cgroups"
-  revision = "4994991857f9b0ae8dc439551e8bebdbb4bf66c1"
+  name = "github.com/docker/docker"
+  revision = "f76d6a078d881f410c00e8d900dcdfc2e026c841"
 
 [[override]]
   name = "github.com/docker/distribution"
-  revision = "0d3efadf0154c2b8a4e7b6621fff9809655cc580"
+  revision = "83389a148052d74ac602f5f1d62f86ff2f3c4aa5"
+
+[[override]]
+  name = "k8s.io/apimachinery"
```

Comment:
Are those dependencies added because of cnab-to-oci?

Reply:
It's because of `docker/docker`.